spec/routes/web/inference_playground_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require_relative "spec_helper"
+
+RSpec.describe Clover, "inference-playground" do
+  let(:user) { create_account }
+
+  let(:project) { user.create_project_with_default_policy("project-1") }
+  let(:project_wo_permissions) { user.create_project_with_default_policy("project-2", default_policy: nil) }
+
+  describe "feature enabled" do
+    before do
+      project.set_ff_inference_ui(true)
+      project_wo_permissions.set_ff_inference_ui(true)
+      login(user.email)
```

Comment:
No need to give email since `create_account` use same email constant 
```suggestion
      login
```